test/integration/scheduler_perf/scheduler_bench_test.go

```diff
@@ -43,6 +43,7 @@ func BenchmarkScheduling(b *testing.B) {
 		{nodes: 100, existingPods: 1000, minPods: 100},
 		{nodes: 1000, existingPods: 0, minPods: 100},
 		{nodes: 1000, existingPods: 1000, minPods: 100},
+		{nodes: 5000, existingPods: 1000, minPods: 100},
```

Comment:
please change minPods to at least 1000. In small clusters, we may not be able to run 1000 pods, but in a cluster of 1000 or more nodes, we certainly can. 100 pods are scheduled in ~1 second which is too short for a meaningful benchmark.

Reply:
Ok. I bumped up `minPods` to `1000` in all 5k test cases.